wazo_calld/plugins/dial_mobile/services.py

```diff
@@ -281,3 +281,29 @@ def cancel_push_mobile(self, call_id):
             pending_push.tenant_uuid,
             pending_push.user_uuid,
         )
+
+    def remove_pending_push_mobile(self, call_id):
+        self._pending_push_mobile.pop(call_id, None)
+
+    def is_push_answered_by_mobile(self, push_call_id, call_id, endpoint, user_uuid):
```

Comment:
The name `answer` is misleading, because it does not need to be the mobile that answers the caller, but it only needs the mobile to have registered after a push notification.

Suggestion: `is_push_received_by_mobile` or `has_pending_push`+ `is_mobile_registered`

Reply:
This is only for answered calls. We check if the device that entered the bridge is registered as mobile and owned by the user being called.